### PR TITLE
Fix issue w StateReadState eth addr

### PR DESCRIPTION
**Pull request by Schwartz10**

### CODE REVIEW

File: src/components/InputV2/Address.tsx [hunk 1]
```diff
@@ -34,6 +43,28 @@ export const AddressInput = ({
...
+  )
+
+  // Communicate delegated address to parent component
+  useEffect(
+    () => setDelegated(delegatedFromEth),
+    [setDelegated, delegatedFromEth]
+  )
+
+  // Store any converted value for display
+  const converted = delegatedFromEth || ethFromDelegated
```
Comment (Schwartz10) [inline src/components/InputV2/Address.tsx]: Here i think we should change the text based on which way we're going:
```
let suffix = ''

if (delegatedFromEth) suffix = "Converts into ${delegatedFromEth}"
else if (ethFromDelegated) suffix = "Represented by ${ethFromDelegated}"
```
I'm not sure the difficulty on this, but ideally, `delegatedFromEth` and `ethFromDelegated` would be `AddressLink` to the explorer

Comment (navFooh) [reply src/components/InputV2/Address.tsx]: Done!

Comment (navFooh) [reply src/components/InputV2/Address.tsx]: Ooh wait I didn't do the AddressLink

Comment (navFooh) [reply src/components/InputV2/Address.tsx]: Done!

**Pull request merged**